setup.py

```diff
@@ -43,6 +29,24 @@ def get_version(rel_path):
     return "{}.{}.{}".format(major_version, minor_version, patch_version)
 
 
+def get_install_requires():
+    install_requires = [
+        'neo4j',
+        'praatio~=5.0',
+        'textgrid',
+        'conch_sounds',
+        'librosa',
+        'influxdb',
+        'tqdm',
+        'requests',
+        'scipy~=1.12.0',
+        'setuptools' # only added to work with praatio
+    ]
+    if os.name == 'nt':
+        install_requires.append('pywin32')
```

Comment:
I forget at this point what pywin32 is necessary for, is it just needed for neo4j installation?  I'll try to look into the work required to make a neo4j conda forge feedstock, since that would simplify installation a lot.

Reply:
Thanks for the review! Pywin32 is used when installing neo4j as a service on Windows. It is for running the command as administrator (for the permission popup, I think). It would certainly be great if conda forge can provide an easier approach.

Reply:
Hi @mmcauliffe,
I think it's quite uncommon for entire databases to be available on conda-forge (as it's mainly used for package management). However, I was considering setting up Docker to manage these databases. Do you have any suggestions regarding this approach?

Reply:
Right, so there is the [postgresql feedstock](https://github.com/conda-forge/postgresql-feedstock) and [influxdb feedstock](https://github.com/conda-forge/influxdb-feedstock), but I don't know how much work it would take to actually get a neo4j-feedstock working correctly on all platform, particularly if the windows-service installation is a required aspect or if it can be run by a user in more recent versions.  

I think docker as a first step for covering the necessary prereqs is a good idea, you can have it auto build and publish for every new release via github actions like MFA does: [.github/workflows/publish_docker.yml](https://github.com/MontrealCorpusTools/Montreal-Forced-Aligner/blob/main/.github/workflows/publish_docker.yml) with its [Dockerfile](https://github.com/MontrealCorpusTools/Montreal-Forced-Aligner/blob/main/Dockerfile).  Note particularly [line 16](https://github.com/MontrealCorpusTools/Montreal-Forced-Aligner/blob/main/Dockerfile#L16) of it, that's the code that's responsible for starting the postgresql server that MFA uses.

Reply:
Ohh, I see what you mean now! I misunderstood what a feedstock is. I think it would be a great idea to have a Neo4j feedstock, and I will look into that as well. Meanwhile, I will set up Docker as a first step. Thanks for the clarification and guidance!